includes/functions.php

```diff
@@ -400,6 +400,26 @@ function convertkit_get_form_editor_url() {
 
 }
 
+/**
+ * Helper method to return the URL the user needs to visit on the ConvertKit app to create a new Landing Page.
+ *
+ * @since   2.5.5
+ *
+ * @return  string              ConvertKit App URL
+ */
+function convertkit_get_new_landing_page_url() {
+
+	return add_query_arg(
+		array(
+			'utm_source'  => 'wordpress',
+			'utm_term'    => get_locale(),
```

Comment:
I'm a bit confused why we're sending the local as the utm_term - I tried searching Slack for info but turned up nothing.

Reply:
This is the case for most of the URLs for ConvertKit, so Brendan can track the WordPress locale to determine if we need to offer translations for the Plugin in the future.